app/layout.tsx

```diff
@@ -3,7 +3,7 @@ import './globals.css';
 import {Theme} from '@radix-ui/themes';
 import type {Metadata} from 'next';
 import {Rubik} from 'next/font/google';
-import Script from 'next/script';
+import PlausibleProvider from 'next-plausible';
```

Comment:
What is the advantage of having this instead of the script?

Reply:
send events dynamically through a hook, nice TS and local DX experience